<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplaySettingsDock</class>
 <widget class="QDockWidget" name="DisplaySettingsDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display Settings</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::FieldsStayAtSizeHint</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="moleculeTypeLabel">
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="moleculeTypeComboBox">
      <item>
       <property name="text">
        <string>Ball And Stick</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Stick</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Space Filling</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QCheckBox" name="showHydrogensCheckBox">
      <property name="text">
       <string>Show Hydrogens</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QCheckBox" name="showBondOrderCheckBox">
      <property name="text">
       <string>Show Bond Order</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QCheckBox" name="showPymolSESCheckBox">
      <property name="text">
       <string>Show Pymol SES</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QCheckBox" name="showPymolSASCheckBox">
      <property name="text">
       <string>Show Pymol SAS</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
